bike-shop/src/stage3-methods.js

```diff
@@ -8,7 +34,23 @@ function Frame() {
 
 function Tire() {
```

Comment:
since this function isn't being used I would just delete the function until you have some functionality to add to it.